assets/thanos-ruler/thanos-ruler.yaml

```diff
@@ -1,6 +1,8 @@
 apiVersion: monitoring.coreos.com/v1
 kind: ThanosRuler
 metadata:
+  annotations:
+    operator.prometheus.io/controller-id: openshift-monitoring/prometheus-operator
```

Comment:
I thought thanos ruler was managed by UWM prom-operator.

Reply:
i see it's deployed in `openshift-user-workload-monitoring`

Reply:
(we can wait for the tests to fail if you want :))

Reply:
Yep you are right, I have changed it by error

Reply:
fix it